### PR TITLE
Tweak language describing PM Group

### PR DESCRIPTION
**Pull request by calebamiles**
Soften language used when describing the PM Group

### CODE REVIEW

**Comment by calebamiles** [comment]
**cc: @philips**

File: sig-pm/README.md [hunk 1]
```diff
@@ -1,54 +1,29 @@
...
 
-The primary goal of the PM group is Product Management (PdM), considering Kubernetes as a product. Also, SIG PM covers the areas of Project Management (PjM) and Program Management (PgM) of Kubernetes.
+### Common activities
+- Collecting and generalizing user feedback to help drive project direction and priorities: delivering on user needs while enforcing vendor neutrality
+- Supporting collaboration across the community by working to improve the communication of roadmap and workload of other [Special Interest Groups](https://github.com/kubernetes/community#special-interest-groups-sig-and-working-groups)
+- Supporting the continued effort to improve the stability and extensibility of Kubernetes Project
+- Supporting the marketing and promotion of the Kubernetes Project through the [CNCF](https://www.cncf.io/)
+- Working with the [Kubernetes Release Team](https://github.com/kubernetes/community/tree/master/contributors/devel/release) to continually ensure a high quality release of the Kubernes Project
```
Comment (stp-ip) [inline sig-pm/README.md]: Kubernes -> Kubernetes

**Comment by stp-ip** [comment]
+1

File: sig-pm/README.md [hunk 2]
```diff
@@ -2,7 +2,7 @@
...
+The Kubernetes PM Group focuses on aspects of product management, such as the qualification and successful management of user requests, and aspects of project and program management such as he continued improvement of the processes used by the Kubernetes community to maintain the Kubernetes Project itself. Besides helping to discover both what to build and how to build it, the PM Group also helps to try and keep the wheels on this spaceship we are all building together; bringing together people who think about Kubernetes as both a vibrant community of humans and technical program is another primary focus of this group. Members of the Kubernetes PM Group can assume [certain additional](https://github.com/kubernetes/community/blob/master/project-managers/README.md) responsibilities to help maintain the Kubernetes Project itself. It is also important to remember that the role of managing an open source project is very new and largely unscoped for a project as large as Kubernetes; we are learning too and we are excited to learn how we can best serve the community of users and contributors.
```
Comment (stp-ip) [inline sig-pm/README.md]: Suggestion:
The Kubernetes PM Group focuses on aspects of product management, such as the qualification and successful management of user requests, and aspects of project and program management such as the continued improvement of processes used by the Kubernetes community to maintain the project itself. Further goals are helping to discover what to build and how to build it. The PM Group also helps to try and keep the wheels on this spaceship we are all building together. Another primary focus is bringing together people who think about Kubernetes as a vibrant community of humans, but also as a technical program. Members of the Kubernetes PM Group can assume [certain additional](https://github.com/kubernetes/community/blob/master/project-managers/README.md) responsibilities to help maintain the Kubernetes Project itself. It is also important to remember that the role of managing an open source project is very new and largely unscoped for a project as large as Kubernetes; we are excited to learn and keep learning how we can best serve the community of users and contributors.

Comment (apsinha) [inline sig-pm/README.md]: LGTM.

Comment (calebamiles) [inline sig-pm/README.md]: Thanks for the suggestion @stp-ip but I think I'll keep the copy largely unchanged because I was aiming for `Besides helping to discover both what to build and how to build it` to relate to the previous sentence talking about Product/Project/Program Management rather than standing as it's own thought point and I wanted that thought connected to the thought of the SIG as a rallying point for people such that  `the PM Group also helps to try and keep the wheels on this spaceship we are all building together` and `bringing together people who think about Kubernetes as both a vibrant community of humans and technical program is another primary focus of this group` formed a single sentence. It's a bit wordy but I think generally ok...at least good enough to merge

Comment (stp-ip) [inline sig-pm/README.md]: That's fine, but please open another PR to fix: "program management such as he continued improvement" -> "program management such as the continued improvement" (he -> the)
Then I can live with my suggestion not being used ;)

**Comment by idvoretskyi** [comment]
Thanks for softening, @calebamiles! A few suggestions from me are in a new PR - https://github.com/kubernetes/community/pull/364

PS. Hope, it will be the last iteration of the document.
